articles/dns/dns-faq-private.md

```diff
@@ -38,6 +38,14 @@ Yes. You must have write operation permission on the virtual networks and the pr
 
 Yes. If you delete a virtual machine within a linked virtual network with autoregistration enabled, the registered records are automatically deleted.
 
+## I've reconfigured the OS in my virtual machine to have a new hostname or static IP address but I don't see that reflected in the private zone.  Why not?
+
+The private zone's records are populated by the Azure DHCP servers; client registration messages are ignored.  Thus, if you have disabled DHCP client support in the VM by configuring a static IP address, changes to the hostname or static IP in the VM will not be reflected in the zone.
```

Comment:
Please replace "servers" in "Azure DHCP servers" to "service" i.e. "Azure DHCP service"